test/fixtures/vcr_cassettes/actions/pulp3/remove_orphans/orphans_are_removed.yml

```diff
@@ -2,15 +2,15 @@
 http_interactions:
```

Comment:
Did this test actually have to be re-recorded?  And if so do you know why it increased in size so much?

Reply:
One of the tests failed with that particular VCR tape listed as out of date.

Reply:
oh i see its a different test than the one above